app/views/sessions/new.html.erb

```diff
@@ -15,11 +15,6 @@
         <span><%= t ".remember_me" %></span>
       <% end %>
 
-      <%= f.label :remember_me, class: "checkbox inline" do %>
```

Comment:
Why remove this ?

Reply:
@thanhhungchu95 I've double this last time.